### DISCUSSION TITLE
Pushover notifications

### DISCUSSION THREAD

**Discussion by iFredOS** [containrrr/watchtower#645] [Feature Requests] [answered]
Can you add this is very useful and free notifications service, Pushover? This is very easy to implement. I'm not (yet) a programmer but I created my own bash script for my local custom notifications and works flawlessly.

https://pushover.net

Thank you very much in advance!

**Comment by simonaronsson** [comment, marked as answer]
Hi,

We already support pushover using Shoutrrr. [Here](https://containrrr.dev/shoutrrr/services/pushover/)'s a link to the docs.

Best,
Simme
